src/sentry/api/endpoints/organization_config_repositories.py

```diff
@@ -13,11 +13,12 @@ def get(self, request, organization):
         providers = []
         for provider_id in provider_bindings:
             provider = provider_bindings.get(provider_id)(id=provider_id)
-            providers.append({
-                'id': provider_id,
-                'name': provider.name,
-                'config': provider.get_config(),
-            })
+            if provider.name != 'Bitbucket' or bool(organization.flags.early_adopter):
```

Comment:
i think we normally do this by creating a custom feature flag and then adding it here: https://github.com/getsentry/getsentry/blob/master/getsentry/features.py#L232